### DISCUSSION TITLE
Sudo doesn't work.

### DISCUSSION THREAD

**Discussion by spieler0** [EssentialsX/Essentials#5140] [Q&A]
When I try sudoing a player, it says "You cannot sudo (player)". I am op myself and have the sudo permission and the player im trying to sudo isn't op and doesnt have the essentials.sudo.exempt permission. What did I do wrong?

**Comment by Evidentsinger14** [comment]
can you run `/ess debug`, try to sudo the player, run `/ess debug` again, and then run `/ess dump log` and send the link provided to you? You can also join the [Support Discord](https://discord.gg/PHpuzZS) to get help.

**Reply by spieler0** [reply]
https://essentialsx.net/dump.html?id=dd1daa58c29c4ab6a7eb81190fe810a3

**Reply by Evidentsinger14** [reply]
> [17:05:25] [Server thread/INFO]: [Essentials] checking if Leafyr has essentials.sudo.exempt - true

they do have the sudo.exempt permission. i would double check if they're opped or not, and check if they have any wildcard permissions. 
`essentials.*`, `essentials.sudo.*`, `essentials.sudo.exempt`, or `*` directly would all include this permission.

**Reply by spieler0** [reply]
Im using Luckperms and I dont think i got essentials.sudo.exempt there.

**Reply by Evidentsinger14** [reply]
you can run `/lp user <user> permission check essentials.sudo.exempt` to see where it's coming from. It should list all groups they inherit, and which one it may be from.
